go/chat/errors.go

```diff
@@ -80,6 +80,18 @@ func (e EphemeralUnboxingError) Error() string {
 
 //=============================================================================
 
+type EphemeralKeyError struct{}
```

Comment:
Maybe name this type something public-specific?

Reply:
sgtm!